akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -59,10 +60,15 @@ private[akka] object RemoteActorRefProvider {
     }
 
     when(WaitTransportShutdown) {
-      case Event((), _) ⇒
+      case Event(Done, _) ⇒
         log.info("Remoting shut down.")
         systemGuardian ! TerminationHookDone
         stop()
+
+      case Event(Status.Failure(ex), _) ⇒
+        log.error(ex, "Remoting shut down with error")
```

Comment:
@patriknw in general it's not limited to `AskTimeoutException`, it can be as well `IllegalArgumentException` (from `ask`) or any other, which got piped.
error was used as initil variant as `Remoting` uses `notifyError` and publishes `RemoteErrorEvent`, when shutdown goes wrong.